Sources/Sentry/SentrySDK.m

```diff
@@ -137,24 +137,29 @@ + (void)setStartInvocations:(NSUInteger)value
 
 + (void)startWithOptions:(SentryOptions *)options
 {
+    [SentryLog configure:options.debug diagnosticLevel:options.diagnosticLevel];
+
     // We accept the tradeoff that the SDK might not be fully initialized directly after
     // initializing it on a background thread because scheduling the init synchronously on the main
     // thread could lead to deadlocks.
```

Comment:
`m`: I think that comment should stay directly above `[SentryThreadWrapper onMainThread:^{`